features/kata/operator.feature

```diff
@@ -0,0 +1,12 @@
+Feature: kata operator related tests
+  Background:
+  Given valid cluster type for kata tests exists
+
+  # @author valiev@redhat.com
+  # @case_id OCP-41813
+  @admin
+  @destructive
+  Scenario: operator successfully installed
```

Comment:
@valtair1891 perhaps change the scenario to `kata operator successfully installed`

Reply:
@pruan-rht you are right, it's done